### ISSUE TITLE
Need new algorithm to analyse question papers

### ISSUE THREAD

**Issue by AnirudhDaya** [FAHADPN/Syllabus-condenser#5]
The current algorithm only analyses a specific set of question papers, we need to make it more generic and applicable to all.

**Comment by FAHADPN** [comment]
Yes, We should plan to make a more inclusive algorithm to accommodate all the types of queries. I hope to have an open source contribution towards our project

**Comment by AnirudhDaya** [comment]
I am creating a new branch for this issue so it doesn't affect our main branch
